### PR TITLE
Implement POC

### PR DESCRIPTION
**Pull request by vvkin**

### CODE REVIEW

File: src/plugins/user/user.dao.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    const user = await this.pg<User>('users').where({ email }).first();
+    return user;
+  }
+
+  async findByLogin(login: string): Promise<User | undefined> {
+    const user = await this.pg<User>('users').where({ login }).first();
+    return user;
+  }
+
+  async findByEmailOrLogin(criterion: string): Promise<User | undefined> {
```
Comment (nosovk) [inline src/plugins/user/user.dao.ts:16]: в таких прокладках точно не нужен async await
```typescipt
  findByEmailOrLogin(criterion: string): Promise<User | undefined> {
  return this.pg<User>('users')
      .where({ login: criterion })
      .orWhere({ email: criterion })
      .first();
  }
```

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -1,29 +1,47 @@
...
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
     steps:
       - uses: actions/checkout@v2
-      - name: Cache node_modules
-        uses: actions/cache@v2
-        id: cache-node-modules
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
```
Comment (nosovk) [inline .github/workflows/ci.yml]: v3?

Comment (vvkin) [reply .github/workflows/ci.yml]: Fixed

**Comment by vvkin** [comment]
Close as it's not relevant anymore. It was opened only for the entire project review.

**Pull request closed without merging**